commit ba837e02d62c32245569124e050a654335c5ac84
Author: Boris Brezillon <boris.brezillon@free-electrons.com>
Date:   Tue Nov 22 12:45:28 2016 -0800

    clk: bcm2835: Fix ->fixed_divider of pllh_aux
    
    There is no fixed divider on pllh_aux.
    
    Signed-off-by: Boris Brezillon <boris.brezillon@free-electrons.com>
    Signed-off-by: Eric Anholt <eric@anholt.net>
    Reviewed-by: Eric Anholt <eric@anholt.net>
    Signed-off-by: Stephen Boyd <sboyd@codeaurora.org>
    (cherry picked from commit f2a46926aba1f0c33944901d2420a6a887455ddc)

diff --git a/drivers/clk/bcm/clk-bcm2835.c b/drivers/clk/bcm/clk-bcm2835.c
index 21e2a538ff0d..a99ccf9f056d 100644
--- a/drivers/clk/bcm/clk-bcm2835.c
+++ b/drivers/clk/bcm/clk-bcm2835.c
@@ -1607,7 +1607,7 @@ static const struct bcm2835_clk_desc clk_desc_array[] = {
 		.a2w_reg = A2W_PLLH_AUX,
 		.load_mask = CM_PLLH_LOADAUX,
 		.hold_mask = 0,
-		.fixed_divider = 10),
+		.fixed_divider = 1),
 	[BCM2835_PLLH_PIX]	= REGISTER_PLL_DIV(
 		.name = "pllh_pix",
 		.source_pll = "pllh",
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1150_ba837e02d62c32245569124e050a654335c5ac84.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1150_ba837e02d62c32245569124e050a654335c5ac84.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
